tests/unit/parse-test.js

```diff
@@ -9,7 +9,7 @@ describe('Parse', () => {
     describe('literals', () => {
       it('handles language subtags', () => {
         let base = 'https://www.wikidata.org/wiki/Special:EntityData/Q2005.ttl'
-        let mimeType = 'text/turtle'
+        let mimeType = 'text/turtle; charset="UTF-8"'
```

Comment:
gotcha, yes this would be better - I'll add some more explicit tests when in an IDE, this was just a quick fix and check to unblock somebody - but it turns out solid server is test failing any way so wasn't fruitful :(